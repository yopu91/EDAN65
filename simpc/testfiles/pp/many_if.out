g(

if (
a)

a = a
if (
b)

b = b
return 
0
