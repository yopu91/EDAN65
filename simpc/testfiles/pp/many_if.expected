int g()
	if
		a
		a=b
	if
		b
		b=a
	return
		a
